shared/settings/index.js

```diff
@@ -10,7 +12,11 @@ class Settings extends Component {
   static parseRoute () {
     return {
       componentAtTop: {title: 'Settings'},
-      subRoutes: {devMenu},
+      subRoutes: {
+        devMenu,
+        deleteConfirm: DeleteConfirm,
+        removeDevice: RemoveDevice,
```

Comment:
Makes sense, thanks. Reading this reminded me of the other reason behind this: want to be able to `navigateUp` to the settings screen, not devices page if the user reconsiders revoking their device.